README.md

```diff
@@ -224,8 +224,9 @@ The oauth2-proxy extends your Istio Ingress-Gateway capabilities, to be able to
 
 ```sh
 echo "Installing oauth2-proxy..."
-kustomize build common/oauth2-proxy/overlays/m2m-self-signed/ | kubectl apply -f -
+kustomize build common/oauth2-proxy/overlays/m2m-dex-and-kind/ | kubectl apply -f -
```

Comment:
Here we should probably add a sentence to explain tat there are also other m2m issuers available and link to other files/documentation